Test error for glide install.

```
# github.com/appleboy/gofight/vendor/github.com/labstack/echo
../vendor/github.com/labstack/echo/echo.go:245: cannot use "github.com/appleboy/gofight/vendor/github.com/labstack/gommon/log".New("echo
") (type *"github.com/appleboy/gofight/vendor/github.com/labstack/gommon/log".Logger) as type Logger in field value:
        *"github.com/appleboy/gofight/vendor/github.com/labstack/gommon/log".Logger does not implement Logger (missing Panic method)
../vendor/github.com/labstack/echo/echo.go:558: e.Color.Printf undefined (type *color.Color has no field or method Printf)
../vendor/github.com/labstack/echo/echo.go:562: e.Color.Printf undefined (type *color.Color has no field or method Printf)
FAIL    github.com/appleboy/gofight/example [build failed]
make: *** [example] Error 2
```

as title.

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,11 +15,17 @@ env:
   global:
     - secure: fB4N5c2BfEoaNkpGDM1wPbB9zV4ZhQqsnWZ+wyj8iDWyHlOw3Ungqb3nMEuVQMn3Yeu/PMLeDxd2h2at3+k+8g3IlEdKHS+uMzATVtOwBxwXPjBAB/oROCZPj3QlEFP32Z5AgGK9+tgl/dGjlaRvJ+BTv3YL3UMxPoaxXmi8IN0F2Z48rNgnsdyvq/u3htUGl03YkFZgucVuZ8GGD3TDOoMylcyZ8jYDooQOfoOSh+c1hFj4+mFls4t5Nc9tfgtbhrtzU7SFUKvYe14ryCxVNC7WeyUvS32nrWIS52yhxiyYU0o1umzdVqK/CwKoGh4LetiwmzDMyqZDyzVvcuvAaJ7D2FTU/yJ8t5wb0uWtmCbb6i258voS79LbnCeTxMTBn+h3CFsPKwR+xoiue+2jXIUP4K1+bk8Bj0cSfAwSlY2xkVzB/EpEXhPKlY/cGDMXF5WPpzUyeNdOUMlOiiVJ68V/H7ZwxfWMrgFxmeLHRnrrJtIVPTLdbtigf0WFew48+WELo9lv6SVYy7Q+Y46A46nNBDKqbJtHT/AhHPgqmi0Bt+5NZ/q0DgQd7og/ISGy0tbFRUUcKNFEumGjndtv7eY6GCvMIyEPZXclzPdNrPqAdWDqNeP6cwBCljx7Fve50QDe/3KAwMLEMvux9Qn0KcgIMNCrhfnAGJqEKx8BZvc=
 
+before_install:
+  - mkdir -p $GOPATH/bin
+  - curl https://glide.sh/get | sh
+
 install:
   - export GO15VENDOREXPERIMENT=1
   - go get github.com/mattn/goveralls
+  - make install
 
 script:
+  - make test
   - make docker_test
   - $(go env GOPATH | awk 'BEGIN{FS=":"} {print $1}')/bin/goveralls -coverprofile=.cover/coverage.txt
     -service=travis-ci -repotoken=$COVERALLS_TOKEN

--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@ export PROJECT_PATH = /go/src/github.com/appleboy/gofight
 all: install test
 
 install:
-	go get -t -d -v ./...
+	glide install
 
 example:
 	cd example && go test -v -cover .

--- a/glide.lock
+++ b/glide.lock
@@ -1,0 +1,70 @@
+hash: be54b33c19e74d67131a7cbfd15ce10ef31331adde96098f131acb97f4abaf79
+updated: 2017-01-09T23:00:38.964928246+08:00
+imports:
+- name: github.com/davecgh/go-spew
+  version: 04cdfd42973bb9c8589fd6a731800cf222fde1a9
+  subpackages:
+  - spew
+- name: github.com/gin-gonic/gin
+  version: e2212d40c62a98b388a5eb48ecbdcf88534688ba
+  subpackages:
+  - binding
+  - render
+- name: github.com/golang/protobuf
+  version: 2402d76f3d41f928c7902a765dfc872356dd3aad
+  subpackages:
+  - proto
+- name: github.com/gorilla/context
+  version: 08b5f424b9271eedf6f9f0ce86cb9396ed337a42
+- name: github.com/gorilla/mux
+  version: 757bef944d0f21880861c2dd9c871ca543023cba
+- name: github.com/julienschmidt/httprouter
+  version: 8a45e95fc75cb77048068a62daed98cc22fdac7c
+- name: github.com/labstack/echo
+  version: 8d504c1b699c757b267255c53b3e5219f9974abc
+- name: github.com/labstack/gommon
+  version: c8c07aa087422c146b86d2f79556b450a8e630a1
+  subpackages:
+  - color
+  - log
+- name: github.com/manucorporat/sse
+  version: ee05b128a739a0fb76c7ebd3ae4810c1de808d6d
+- name: github.com/mattn/go-colorable
+  version: d228849504861217f796da67fae4f6e347643f15
+- name: github.com/mattn/go-isatty
+  version: 30a891c33c7cde7b02a981314b4228ec99380cca
+- name: github.com/pmezard/go-difflib
+  version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
+  subpackages:
+  - difflib
+- name: github.com/stretchr/testify
+  version: 2402e8e7a02fc811447d11f881aa9746cdc57983
+  subpackages:
+  - assert
+- name: github.com/tylerb/graceful
+  version: d37e108c89765a8e307f15b8fb2ecd10575da6bb
+- name: github.com/valyala/fasttemplate
+  version: 3b874956e03f1636d171bda64b130f9135f42cff
+- name: golang.org/x/crypto
+  version: c3b1d0d6d8690eaebe3064711b026770cc37efa3
+  subpackages:
+  - acme
+  - acme/autocert
+- name: golang.org/x/net
+  version: f315505cf3349909cdf013ea56690da34e96a451
+  subpackages:
+  - context
+  - context/ctxhttp
+- name: golang.org/x/sys
+  version: d75a52659825e75fff6158388dddc6a5b04f9ba5
+  subpackages:
+  - unix
+- name: gopkg.in/gin-gonic/gin.v1
+  version: e2212d40c62a98b388a5eb48ecbdcf88534688ba
+- name: gopkg.in/go-playground/validator.v8
+  version: c193cecd124b5cc722d7ee5538e945bdb3348435
+- name: gopkg.in/yaml.v2
+  version: a5b47d31c556af34a302ce5d659e6fea44d90de0
+testImports:
+- name: github.com/buger/jsonparser
+  version: 42b84a5caf12249d724445a0be7df0272f30b9a9

--- a/glide.yaml
+++ b/glide.yaml
@@ -1,0 +1,12 @@
+package: github.com/appleboy/gofight
+import:
+- package: github.com/gorilla/mux
+- package: github.com/julienschmidt/httprouter
+- package: github.com/stretchr/testify
+  subpackages:
+  - assert
+- package: gopkg.in/gin-gonic/gin.v1
+- package: github.com/labstack/echo
+  version: ^3.0.3
+testImport:
+- package: github.com/buger/jsonparser

